config/index.js

```diff
@@ -7,7 +7,7 @@ module.exports = {
     index: path.resolve(__dirname, '../docs/index.html'),
     assetsRoot: path.resolve(__dirname, '../docs'),
     assetsSubDirectory: 'static',
-    assetsPublicPath: '/',
```

Comment:
Here I tried to reproduce the assets path as here : https://apertureless.github.io/vue-cookie-law/.
So I had to change this config.